devops/github-actions/s3-storage-cicd/.github/workflows/dev.yml

```diff
@@ -0,0 +1,20 @@
+name: Create or update deployment using Github Storage
+
+on:
+  push:
+    branches:
+      - dev
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: jakejarvis/s3-sync-action@master
+        with:
+          args: --exclude '.venv/*' --exclude '.git/*'
+        env:
+          AWS_S3_BUCKET: ${{ secrets.AWS_S3_BUCKET }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          DEST_DIR: "/dev"
```

Comment:
I don't understand why we're syncing a folder to s3, isn't this stepping on the toes of the `prefect deployment build` command?

Reply:
That's a good point. I would have expected a repo checkout step, then a `prefect deployment build` step.